packages/site/src/pages/ProposalDetail/timeline/index.js

```diff
@@ -87,6 +95,17 @@ function constructProposalProcessItem(item, proposalDetail) {
           title: "Value",
           value: <Balance value={value} />,
         },
+        {
```

Comment:
Treasury proposal don't have `pending payout` timeline item. We should add this item to `bounties` and `child bounties`.